test/integration/consts_test.go

```diff
@@ -1,17 +1,21 @@
 package integration
 
-import "time"
+import (
+	"time"
+
+	"github.com/kong/kubernetes-ingress-controller/v3/test/integration/consts"
+)
 
 // -----------------------------------------------------------------------------
 // Testing Timeouts
 // -----------------------------------------------------------------------------
 
 const (
 	// waitTick is the default timeout tick interval for checking on ingress resources.
-	waitTick = time.Second * 1
+	waitTick = consts.WaitTick
```

Comment:
Why not use `consts.WaitTick` directly? Did you want to keep this PR smaller? Or other rationale

Reply:
It allows us to perform a migration in a less breaking manner: we don't need to replace all `waitTick` occurrences in `test/integration/` thanks to this alias.

When we get to a point where all integration tests have migrated to test suite introduced in this PR then we'll be able to refactor this and get rid of this `const` iteratively.